src/app/public/modules/lookup/lookup.component.scss

```diff
@@ -93,6 +105,11 @@ sky-input-box .sky-lookup {
 
 @include sky-theme-modern {
 
+  .sky-lookup-inset-search-icon {
+    font-size: 24px;
+    padding: 0 $sky-space-lg;
```

Comment:
I think this should be using `$sky-theme-modern-space-md` since this is for modern theme.

Reply:
Yep, done.